nservicebus/azure-service-bus/error-handling.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Azure Service Bus Error Handling
+reviewed: 2016-08-29
+tags:
+- Azure
+- Cloud
+- Error Handling
+---
+
+## Native Error Handling
```

Comment:
so the title "Azure Service Bus Error Handling" will already be rendered. is this heading necessary?